client/app/start.go

```diff
@@ -184,6 +184,49 @@ func Start(ctx context.Context, cfg Config) (func(context.Context) error, error)
 	}, nil
 }
 
+func CreateApp(ctx context.Context, cfg Config) *App {
```

Comment:
I think this function is pretty much the same code as the Start function above. Can we add a TODO to make sure the Start function uses this function later. To make it consistent

Reply:
@edisonz0718 
You are right. I added To-Do in https://github.com/piplabs/story/pull/157/commits/fdc427d8c6ab021cc10f2df1d6b56882e3821207